shared/test/test_assets.rb

```diff
@@ -48,6 +48,20 @@ def test_assets
     @api.delete_object(sound_filename)
     assert successful?
 
+    # test put asset
+    sound_body = 'stub-sound-contents'
+    sound_filename = 'meow.mp3'
+    response = @api.put_object(sound_filename, sound_body, {'CONTENT_TYPE' => 'json'})
+    actual_sound_info = JSON.parse(response)
+    expected_sound_info = {'filename' => sound_filename, 'category' => 'audio', 'size' => sound_body.length}
+    assert_fileinfo_equal(expected_sound_info, actual_sound_info)
```

Comment:
Additional test ideas:

1. PUT to the same filename again and verify that the object is replaced (rather than failing or making a second object), to show the appropriate idempotent behavior.
2. PUT to a second filename and verify the API can handle different files being uploaded (would catch accidentally hard-coded filename values, etc).